common/notify_stream/src/lib.rs

```diff
@@ -586,50 +593,6 @@ mod tests {
         Ok(())
     }
 
-    /// macOS will follow symlink files
-    #[tokio::test]
-    #[cfg(target_os = "macos")]
-    async fn test_watch_symlink_write_after_create_macos() -> io::Result<()> {
```

Comment:
removed for mac?

Reply:
I don't know where this test came from - but it didn't work on macos

Reply:
Lets track all Mac problems in JIRA (create separate tickets). Disabled tests, symptoms, err cause if any.

Reply:
yup will do